6.0/6.0.1/05-networkPolicies.md

```diff
@@ -0,0 +1,107 @@
+<!-- Space: TOOLS -->
+<!-- Parent: Installer -->
+<!-- Title: Network Policies v2 in the installer -->
+<!-- Layout: plain -->
+
+# Network Policies
```

Comment:
I think the style we have used in the document is initial capitalization. So here the heading would be # Network policies

Reply:
Fixed here in the docs as "Network policies"